volumetric/criticalKernelsThinning3D.cpp

```diff
@@ -145,6 +148,8 @@ int main(int argc, char* const argv[]){
   app.add_flag("--verbose,-v",verbose, "Verbose output");
   app.add_option("--exportImage,-o",outputFilenameImg, "Export the resulting set of points to a image compatible with GenericWriter.");
   app.add_option("--exportSDP,-e",outputFilenameSDP, "Export the resulting set of points in a simple (sequence of discrete point (sdp))." );
+  app.add_option("--exportOBJ,-O",outputFilenameOBJ, "Export the resulting set of points in an OBJ file." );
```

Comment:
I think you forget to add it in the doc comment (to be inserted line 54)
A copy past: (I cannot suggest it in the GitHub interface since it is outside the edited area)

  -O,--exportOBJ TEXT                   Export the resulting set of points in an OBJ file.
  -I,--exportInputOBJ TEXT              Export the input set of points in an OBJ file.

Reply:
done in the conflict fix